pkg/kv/kvserver/metrics.go

```diff
@@ -1349,6 +1349,12 @@ The count is emitted by the leaseholder of each range.
 		Measurement: "Intent Resolutions",
 		Unit:        metric.Unit_COUNT,
 	}
+	metaGCUsedClearRange = metric.Metadata{
+		Name:        "queue.gc.info.clearrange",
+		Help:        "Number of successful calls to perform ClearRange operation on full range",
```

Comment:
nit: "Number of successful ClearRange operations during GC".

Let's track failures too.